### PR TITLE
Adds missing TextTrack pages

### PR DESCRIPTION
**Pull request by rachelandrew**
Adds the missing pages to the `TextTrack` WebVTT interface.

Reviewer @jpmedley

### CODE REVIEW

File: files/en-us/web/api/texttrack/mode/index.html [hunk 1]
```diff
@@ -25,19 +25,12 @@
```
Comment (AnilSeervi) [inline files/en-us/web/api/texttrack/mode/index.html:25]: `seoSummary` class can also be removed from this file.

File: files/en-us/web/api/texttrack/removecue/index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+<h3 id="Exceptions">Exceptions</h3>
+
+<dl>
+  <dt>{{domxref("DOMException")}} <code>NotFoundError</code></dt>
+  <dd>Thrown if the given cue is not found in the list of cues.</dd>
+</dl>
+
+<div class="notecard note">
+  <p><strong>Note:</strong> The {{domxref("TextTrackCue")}} interface is an abstract class used as the basis for other cue types such as {{domxref("VTTCue")}}. Therefore, when removing a cue you will be passing in one of the cue types that inherit from <code>TextTrackCue</code>.</p>
```
Comment (jpmedley) [inline files/en-us/web/api/texttrack/removecue/index.html:39]:
```suggestion
  <p><strong>Note:</strong> The {{domxref("TextTrackCue")}} interface is an abstract class used as the parent for other cue interfaces such as {{domxref("VTTCue")}}. Therefore, when removing a cue you will be passing in one of the cue types that inherit from <code>TextTrackCue</code>.</p>
```

Comment (jpmedley) [reply files/en-us/web/api/texttrack/removecue/index.html:39]: Please copy this rewrite to the interface summary.

File: files/en-us/web/api/texttrack/removecue/index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+</div>
+
+<h2 id="Examples">Examples</h2>
+
+<p>In the following example a cue added to a video text track using <code>addCue()</code>, then removed using <code>removeCue</code>.</p>
```
Comment (jpmedley) [inline files/en-us/web/api/texttrack/removecue/index.html:44]:
```suggestion
<p>In the following example a cue is added to a video text track using <code>addCue()</code>, then removed using <code>removeCue</code>.</p>
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/texttrack/index.html [hunk 4]
```diff
@@ -13,9 +13,9 @@
 ---
 <div>{{APIRef("WebVTT")}}</div>
 
-<div id="interface_description">
 <p>The <code>TextTrack</code> interface—part of the API for handling WebVTT (text tracks on media presentations)—describes and controls the text track associated with a particular {{HTMLElement("track")}} element.</p>
-</div>
+
+<p><strong>Note:</strong> The {{domxref("TextTrackCue")}} interface is an abstract class used as the parent for other cue interfaces such as {{domxref("VTTCue")}}. Therefore, when adding or removing a cue you will be passing in one of the cue types that inherit from <code>TextTrackCue</code>.</p>
```
Comment (AnilSeervi) [inline files/en-us/web/api/texttrack/index.html:18]: `cue types` can be `cue interfaces`

**Review by jpmedley — APPROVED** [review]

**Pull request merged**